google/fingerprinters/web/scripts/updater/community/craftcms/app/env

```diff
@@ -0,0 +1,28 @@
+# Read about configuration, here:
+# https://craftcms.com/docs/4.x/config/
+
+# The application ID used to to uniquely store session and cache data, mutex locks, and more
+CRAFT_APP_ID=
+
+# The environment Craft is currently running in (dev, staging, production, etc.)
+CRAFT_ENVIRONMENT=production
+
+# The secure key Craft will use for hashing and encrypting data
+CRAFT_SECURITY_KEY=root
+
+# Database connection settings
+CRAFT_DB_DRIVER=mysql
+CRAFT_DB_SERVER=mysql
+CRAFT_DB_PORT=3306
+CRAFT_DB_DATABASE=dev_craftcms
+CRAFT_DB_USER=craftcms
+CRAFT_DB_PASSWORD=SecretPassword
+CRAFT_DB_SCHEMA=public
+CRAFT_DB_TABLE_PREFIX=
+
+# General settings (see config/general.php)
+DEV_MODE=false
+ALLOW_ADMIN_CHANGES=false
+DISALLOW_ROBOTS=false
```

Comment:
This way you can keep only one env file and not have to do differentiate between version 3 and the other ones.

```suggestion
# The application ID used to to uniquely store session and cache data, mutex locks, and more
CRAFT_APP_ID=
APP_ID=

# The environment Craft is currently running in (dev, staging, production, etc.)
CRAFT_ENVIRONMENT=production
ENVIRONMENT=production

# The secure key Craft will use for hashing and encrypting data
CRAFT_SECURITY_KEY=root
SECURITY_KEY=root

# Database connection settings
CRAFT_DB_DRIVER=mysql
DB_DRIVER=mysql
CRAFT_DB_SERVER=mysql
DB_SERVER=mysql
CRAFT_DB_PORT=3306
DB_PORT=3306
CRAFT_DB_DATABASE=dev_craftcms
DB_DATABASE=dev_craftcms
CRAFT_DB_USER=craftcms
DB_USER=craftcms
CRAFT_DB_PASSWORD=SecretPassword
DB_PASSWORD=SecretPassword
CRAFT_DB_SCHEMA=public
DB_SCHEMA=public
CRAFT_DB_TABLE_PREFIX=
DB_TABLE_PREFIX=

# General settings (see config/general.php)
DEV_MODE=false
ALLOW_ADMIN_CHANGES=false
DISALLOW_ROBOTS=false

CP_TRIGGER=admin
```